service/resource/cloudformationv1/main_stack_test.go

```diff
@@ -97,4 +97,22 @@ func TestMainTemplateExistingFields(t *testing.T) {
 		fmt.Println(body)
 		t.Error("asg element not found")
 	}
+
+	if !strings.Contains(body, "Outputs:") {
+		fmt.Println(body)
+		t.Error("outputs header not found")
+	}
+
+	if !strings.Contains(body, "  WorkersOutput:") {
```

Comment:
The whitespace here worries me a bit. But I guess the indentation is part of the cloud formation yaml?

Reply:
Indeed, I've removed it and added constants for the output identifiers, pls take another look.